lib/managers/KalturaLayoutManager.js

```diff
@@ -820,5 +878,26 @@ class KalturaLayoutManager extends kaltura.KalturaManager {
 		}
 		return outFlavors;
 	}
+
+	drop(request, response, params)
```

Comment:
rename to drop updateEntry and add description to the function also mentioning that it is an @action